libraries/AP_OSD/AP_OSD.h

```diff
@@ -552,6 +588,9 @@ class AP_OSD
         OPTION_DISABLE_CROSSHAIR = 1U<<4,
         OPTION_BF_ARROWS = 1U<<5,
         OPTION_AVIATION_AH = 1U<<6,
+#if AP_OSD_CRSF_PANELS_ENABLED
+        OPTION_RF_MODE_ALONG_WITH_LQ = 1U << 20,
```

Comment:
Don't you mean 1U<<7 here?

Reply:
Good catch!! Yeah, 7 makes a lot more sense than 20. Fixed it.